conformance/utils/http/http.go

```diff
@@ -92,6 +92,7 @@ type ExpectedRequest struct {
 // Response defines expected properties of a response from a backend.
 type Response struct {
 	StatusCode    int
+	StatusCodes   []int // alternative to StatusCode, allows multiple acceptable codes
```

Comment:
please add comment that when set StatusCodes takes precedence over StatusCode. 
Ideally we would like ot have only 1 param, can you add TODO to refactor this?